frontend/src/components/documentation/Routes/OAuth/UserData.jsx

```diff
@@ -143,6 +143,18 @@ const UserData = ({ activeLanguage }) => (
           example="ucl-all;ucl-ug;"
           description="A semi-colon delimited string of all UCL intranet groups that the user belongs to"
         />
+        <Cell
+          name="sn"
+          extra="string"
+          example="Mrkvicka"
+          description="Given first name of the user. Can be an empty string."
```

Comment:
I believe it gives the last name of the user not the first name.

Reply:
New description any clearer?